packages/cursorless-org/src/content/enablement-group.mdx

```diff
@@ -0,0 +1,80 @@
+export const meta = {
+  title: "Cursorless Enablement Group",
+  description:
+    "🚀 Help enable the adoption of Cursorless by accelerating its development.",
+};
+
+# {meta.title}
+
+## {meta.description}
+
+Cursorless, an open-source spoken programming language, lets users eliminate the need for a cursor entirely. In other words, software engineers can code at the speed of sound (increased productivity) and all computer users can reduce strain on their wrists to prevent injury (preventative healthcare).
+
+![hats](/big-hats.png)
+
+Cursorless development is user-led. Your support will help Cursorless founder, Pokey Rule, and his team develop the following feature requests as quickly as possible.
+
+- Reduce the learning curve to increase the rate of adoption
+- Launch Cursorless in other IDEs, such as JetBrains, emacs, etc, as well as in a web browser, and even work globally using OCR / accessibility APIs to place hats anywhere on the screen
+- Further improvements to the Cursorless execution engine to advance the state of the art in voice coding.
+
+### 🎯 Goals
+
+Cursorless needs a dedicated, full-time software engineer on staff. This will cost $5,000 USD per month. The Enablement Group guides development and the participation fees contribute to this crucial funding. Cursorless will always be open source.
+
+## 🙌 Join the Cursorless Enablement Group
+
+The Enablement Group consists of stakeholders whose talents, lived experiences, and career experiences bring vital perspectives to the development of Cursorless. Support and input help increase the speed at which new features can be delivered.
+
+## 🥇 Benefits of Joining the Cursorless Enablement Group
+
+{/* <!--- Hey Pokey! Can you do a bit of styling for these 5 which will consider the major benefits. I have ideas in the figmaa wireframe if helpful! --> */}
```

Comment:
I do think even with the image / emojis we could use a bit more visual interest. Shall I take a crack at making these into boxes like the Figma, or really doesn't matter?

Reply:
@pokey hey - I do think you could stylize if you have the time. Whether you use the Figma wireframe or have a way to make them "stand out" more with more common markdown code, open to that.
I'll say that getting the program launched soon is attractive to me at this point